qemu/tests/cfg/numa_opts.cfg

```diff
@@ -72,15 +75,19 @@
         - defaults:
           extra_params += " -numa node -numa node -numa node"
           # nodes are 8MB-aligned:
-          numa_node0_cpus = "0 3"
+          numa_node0_cpus = "0 1 2"
+          numa_node1_cpus = "3 4 5"
+          numa_node2_cpus = ""
+          Host_RHEL.m6, Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2:
+              numa_node0_cpus = "0 3"
+              numa_node1_cpus = "1 4"
+              numa_node2_cpus = "2 5"
```

Comment:
@MiriamDeng , Can you confirm that this numa_node value is correct ?
+              numa_node0_cpus = "0 3"
+              numa_node1_cpus = "1 4"
+              numa_node2_cpus = "2 5"
I think these cpus should be assigned to two numa node , not three.  and the third numa node should be empty.
I tested the configuration with qemu-kvm-rhev-2.3.0-31.el7 which belong to rhel7.2 version

The test case failed , either.
10:31:55 ERROR| FAIL 6-smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_scsi.up.virtio_net.RHEL.7.3.x86_64.io-github-autotest-qemu.numa_opts.nodes.3.defaults -> TestFail: Wrong CPU set on numa node 0: set([0, 1, 2]). Expected: set([0, 3])